R/html_dependency.R

```diff
@@ -433,6 +443,7 @@ makeDependencyRelative <- function(dependency, basepath, mustWork = TRUE) {
 #' @export
 renderDependencies <- function(dependencies,
   srcType = c("href", "file"),
+  scriptFields = c("src", "integrity", "crossorigin"),
```

Comment:
I don't think there's a need for this argument -- if these fields are provided, it might as well be used.